<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigRules</class>
 <widget class="QWidget" name="ConfigRules">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Service Rules</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_list">
     <item>
      <widget class="QListWidget" name="listWidgetRules">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_buttons">
       <item>
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="text">
          <string>&amp;Add</string>
         </property>
         <property name="icon">
          <iconset theme="list-add">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRemove">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Remove</string>
         </property>
         <property name="icon">
          <iconset theme="list-remove">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_EDIT">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_EDIT">
        <item>
         <widget class="QCheckBox" name="checkBoxEnabled">
          <property name="toolTip">
           <string>Enabled</string>
          </property>
          <property name="text">
           <string>Enabled</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Addresses :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditAddress">
          <property name="toolTip">
           <string>[A-P][1-16] i.e.  E4  E*  *</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="timeExecTime">
          <property name="toolTip">
           <string>Execution Time</string>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>22</hour>
            <minute>38</minute>
            <second>0</second>
            <year>2001</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>HH:mm</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_WEEKDAYS">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_days">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_days">
              <item>
               <widget class="QLabel" name="label_Mon">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>M</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_Tue">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>T</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_Wed">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>W</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_Thu">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>T</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_Frid">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>F</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_Sat">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>S</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_Sun">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>S</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_dayscheck">
              <item>
               <widget class="QCheckBox" name="checkBox_Mon">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Should be executed on Monday</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_Tue">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Should be executed on Tuesday</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_Wed">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Should be executed on Wednesday</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_Thu">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Should be executed on Thursday</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_Fri">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Should be executed on Friday</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_Sat">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Should be executed on Saturday</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_Sun">
                <property name="maximumSize">
                 <size>
                  <width>22</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Should be executed on Sunday</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonClear">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Clear all days</string>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_days">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_Actions">
          <item>
           <widget class="QLabel" name="label_action">
            <property name="text">
             <string>Action:</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QComboBox" name="comboBoxAction">
            <property name="toolTip">
             <string>What action to call</string>
            </property>
            <item>
             <property name="text">
              <string>On</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Off</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Set</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Send</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditAction">
            <property name="toolTip">
             <string>Specific value for action</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_repeat">
            <property name="text">
             <string>Repeat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_repeatTimes">
            <property name="toolTip">
             <string>Repeat this rule x times that day</string>
            </property>
            <property name="maximum">
             <number>1439</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_times">
            <property name="text">
             <string>times, every</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_repeatPeriod">
            <property name="toolTip">
             <string>at every n minutes</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1439</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_mn">
            <property name="text">
             <string>mn</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
